mfes/content/public/content/preview/coreplugins/org.sunbird.player.endpage-1.1/renderer/endpageApp.js

```
@@ -0,0 +1,205 @@
+                var object = {
+                    'config': GlobalContext.config,
+                    'data': undefined,
+                    'metadata': contentMetadata
+                }
+                GlobalContext.config = mergeJSON(AppConfig, contentMetadata);
+                window.globalConfig = GlobalContext.config;
+
+                org.ekstep.contentrenderer.initializePreview(object)
+                EkstepRendererAPI.dispatchEvent('renderer:player:show');
+        } else {
+            if(contentMetadata.identifier && window.parent.hasOwnProperty('onContentNotFound')) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)